frontend/src/views/Login.vue

```diff
@@ -75,6 +76,24 @@ export default {
     },
     onValuesChange() {
       this.backendError = null
+    },
+    async onResend() {
+      if (this.backendError.type !== 'verification-required' || !this.backendError?.email) {
```

Comment:
did you forget a ? in `this.backendError.type` because it could crash if backendError is null

Reply:
Apparently 😅